server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreIT.java

```diff
@@ -150,8 +150,8 @@ private void verifyRestoredData(Map<String, Long> indexStats, boolean checkTotal
         assertHitCount(client().prepareSearch(INDEX_NAME).setSize(0).get(), indexStats.get(statsGranularity) + 1);
     }
 
-    public void testRemoteStoreRestoreFromRemoteSegmentStore() throws IOException {
-        internalCluster().startNodes(3);
+    public void testRemoteSegmentStoreRestore() throws IOException {
+        internalCluster().startDataOnlyNodes(3);
```

Comment:
This is not a fix to flakiness that was observed in https://github.com/opensearch-project/OpenSearch/issues/6248. 

It is just to harden the test so that primary is not the cluster manager node as well.